libs/pipelines/api_pipeline.py

```diff
@@ -45,7 +45,7 @@ def run_on_all_fips_for_intervention(
 
     # Load interventions outside of subprocesses to properly cache.
     get_can_projection.get_interventions()
-    pool = pool or multiprocessing.Pool()
+    pool = pool or multiprocessing.Pool(maxtasksperchild=1)
```

Comment:
Please add _some_ sort of comment describing the motivation for this so that somebody looking at this in the future knows why we set this and what they should worry about if they change it.